apps/docs-snippets2/scripts/test-template.ts

```diff
@@ -8,10 +8,7 @@ afterAll(() => {
   console = { ...consoleBkp };
 });
 
-/**
- * @group node
- * @group browser
- */
+// %TEST_ENVIRONMENT%
```

Comment:
@maschad Regarding the PR description and according to [#3157](https://github.com/FuelLabs/fuels-ts/issues/3157)'s premises:

> GitHub issues must be informed one per line (predictable parsing)

Regexes [here](https://github.com/FuelLabs/github-actions/blob/9de966c6488468852bd7418420c781fc1f3e91da/linear/link-pr-to-linear-issue/src/link-pr-to-linear-issue.ts#L10) and [here](https://github.com/FuelLabs/github-actions/blob/9de966c6488468852bd7418420c781fc1f3e91da/linear/link-pr-to-linear-issue/src/link-pr-to-linear-issue.ts#L16).

Reply:
@nedsalk Should we add logs and basic validations that fail the action so we know something is wrong and needs to be corrected?

Reply:
@arboleya thanks for highlighting that (as an aside I don't think your comment relates to line 11 of this file), I've updated the PR description.

But it seems that Linear had picked it up regardless 🤷🏾

Reply:
> @arboleya 
> @nedsalk Should we add logs and basic validations that fail the action so we know something is wrong and needs to be corrected?

I'm not sure that it's worthwhile to add these checks. I think it's enough that we have it explained in the [readme of the action](https://github.com/FuelLabs/github-actions/tree/master/linear/link-pr-to-linear-issue) and the users should acquaint themselves with it. GitHub also doesn't link issues to PRs when multiple are listed in one line (e.g. `Closes #abc, #def`) and it's up to the developer to know that; once they fix it to link properly in GitHub, the action will pick it up as well.

@maschad You should update the description for the related issues and change it from `Related` to `Related to / Relates to / Part of` because those are the only magic words that our action picks up.

> But it seems that Linear had picked it up regardless 🤷🏾

I just took a look in Linear and this PR isn't linked to the issues. What's weird is that our linear workflow didn't even run on this branch. :thinking:

Reply:
All these explanations can be automatic validations.

Reply:
@arboleya sure, I just thought that it might be overkill. If you feel that it should be done, please create the issue and we can pick it up.